app/controllers/v0/apidocs_controller.rb

```diff
@@ -190,7 +190,6 @@ class ApidocsController < ApplicationController
       Swagger::Responses::UnprocessableEntityError,
       Swagger::Schemas::Address,
       Swagger::Schemas::Appeals::Requests,
-      Swagger::Schemas::Appeals::HigherLevelReview,
```

Comment:
Would love Platform's feedback on whether I properly cleaned up the swagger docs for these removed endpoints. If there's a proper/better way to do it, please let me know!